tui_app.py

```diff
@@ -30,43 +30,57 @@ def set_appimage():
 def control_panel_app():
     # Run TUI.
     while True:
+        options_first = []
         options_default = ["Install Logos Bible Software"]
+        options_main = [f"Install Dependencies", "Download or Update Winetricks", "Run Winetricks"]
+        options_installed = [f"Run {config.FLPRODUCT}," "Run Indexing", "Remove Library Catalog",
```

Comment:
Small typo: Comma after `f"Run {config.FLPRODUCT},"` is inside the quotes rather than outside.